reviews/views.py

```diff
@@ -1,280 +1,139 @@
 from datetime import date
-from collections import OrderedDict
 
 from braces.views import GroupRequiredMixin, LoginRequiredMixin
 from django.conf import settings
 from django.core.exceptions import PermissionDenied
-from django.db.models import Q
 from django.urls import reverse
 from django.utils import timezone
 from django.views import generic
 from django.utils.translation import ugettext_lazy as _
 
-from main.utils import get_reviewers, get_secretary, is_secretary
+from main.utils import get_reviewers, get_secretary
 from proposals.models import Proposal
 from .forms import (DecisionForm, ReviewAssignForm, ReviewCloseForm,
                     ChangeChamberForm)
 from .mixins import (AutoReviewMixin, UserAllowedMixin,
-                     CommitteeMixin, UserOrSecretaryAllowedMixin,
+                     CommitteeMixin,
                      UsersOrGroupsAllowedMixin)
 from .models import Decision, Review
 from .utils import notify_secretary, start_review_route
 
 
-class DecisionListView(GroupRequiredMixin, CommitteeMixin, generic.ListView):
-    context_object_name = 'decisions'
-    template_name = 'reviews/decision_list.html'
+class BaseDecisionListView(GroupRequiredMixin, CommitteeMixin, generic.TemplateView):
+    template_name = 'reviews/ufl_list.html'
     group_required = [
         settings.GROUP_SECRETARY,
         settings.GROUP_GENERAL_CHAMBER,
         settings.GROUP_LINGUISTICS_CHAMBER,
     ]
 
     def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
+        context = super().get_context_data()
```

Comment:
Why does it no longer pass `**kwargs` up the chain? Does TemplateView not support arguments?

Reply:
It does... I don't know why I did that... I'll restore this.

(If I were to guess, I confused this for a bit with ``FancyListApiView.get_context()``, which doesn't accept arguments)